ballista/rust/scheduler/src/state/task_manager.rs

```diff
@@ -60,6 +59,23 @@ pub struct TaskManager<T: 'static + AsLogicalPlan, U: 'static + AsExecutionPlan>
     scheduler_id: String,
     // Cache for active execution graphs curated by this scheduler
     active_job_cache: ExecutionGraphCache,
+    pending_task_queue_size: AtomicUsize,
```

Comment:
I think if data like `active_job_cache` is shared between instances of `TaskManager` then we want to share `pending_task_queue_size` as well. 

```suggestion
    pending_task_queue_size: Arc<AtomicUsize>,
```

And with this `Clone` can be derived.